luxus/style.css

```diff
@@ -0,0 +1,197 @@
+	--jetpack--contact-form--input-padding-left: 16px !important;
+}
+
+.wp-block-jetpack-contact-form input,
+.wp-block-jetpack-contact-form textarea {
+	backdrop-filter: saturate(1.1);
+	background-color: transparent;
+	border-color: var(--wp--custom--input--border--color);
+	border-radius: var(--wp--custom--input--border--radius) !important; /* Requires !important to override local variables with theme variables. */
+	border-width: var(--wp--custom--input--border--width) !important;  /* Requires !important to override local variables with theme variables. */
+	filter: brightness(0.975);
+	font-size: inherit;
+	color: inherit;
+	transition: border-color 0.15s cubic-bezier(.4,0,.2,1);
+}
+
+.jetpack-contact-form .jetpack-field .jetpack-field__input, 
+.jetpack-contact-form .jetpack-field .jetpack-field__textarea {
+	border-color: var(--wp--custom--input--border--color);
+}
+
+.wp-block-jetpack-contact-form input:not(:placeholder-shown),
+.wp-block-jetpack-contact-form textarea:not(:placeholder-shown),
+.wp-block-jetpack-contact-form input:focus,
+.wp-block-jetpack-contact-form textarea:focus {
+	filter: brightness(1);
+	backdrop-filter: saturate(1);
+}
+
+.wp-block-jetpack-contact-form input:focus,
+.wp-block-jetpack-contact-form textarea:focus {
+	border-color: var(--wp--custom--input--focus--border--color);
+}
+
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) select, 
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label)>input, 
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label)>textarea {
+	padding-top: var(--field-padding); /* Remove unnecessary padding adjustment from Jetpack. */
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .animated-label__label {
+	font-size: inherit;
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .animated-label__label {
+	width: auto;
+	padding: 0 4px;
+	transition: transform 0.15s cubic-bezier(.4,0,.2,1), top 0.15s cubic-bezier(.4,0,.2,1), font-size 0.15s cubic-bezier(.4,0,.2,1);
+}
+
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field.has-placeholder~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field:focus~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap .grunion-field:not(:placeholder-shown)~.animated-label__label, 
+.contact-form .is-style-animated .grunion-field-wrap.grunion-field-select-wrap .animated-label__label {
+	background-color: var(--wp--custom--input--color--background) !important;
+	transform: translateY(-11px) translateX(-4px); /* Moves the label out of the field. */
+}
+
+.contact-form label,
+.wp-block-jetpack-contact-form label,
+.jetpack-field-label .rich-text.jetpack-field-label__input {
+	font-weight: 500;
+}
+
+.contact-form__input-error {
+	font-size: var(--wp--preset--font-size--small);
+}
+
+/*
+ * Decorative border for headings with .wave-underline class.
+ */
+:is(h1, h2, h3, h4, h5, h6).wave-underline::after {
```

Comment:
This is causing a jarring hover bug inside the editor where the wavy underline stops underlining


https://github.com/user-attachments/assets/4b87ffb6-c3f5-4d3b-b6b6-42fb825f6258



The editor styles for `.block-editor-block-list__layout` is overriding nearly all of these rules. Maybe the specificity needs increasing here?

Reply:
This is a bit dodgy, isn't it? I'll try to re-do it by creating a custom separator with the wave, should be easier for customers to grasp.

Reply:
I created a custom separator instead and got rid of this dodgy CSS solution :)